src/settings/index.ts

```diff
@@ -538,7 +538,7 @@ export const SHOW_EMOJI_IN_REACTION_PAGE = true;
 export const DEFAULT_ENABLE_JUKEBOX = false;
 export const DEFAULT_SHOW_SHOUTOUTS = true;
 export const DEFAULT_SHOW_REACTIONS = true;
-export const DEFAULT_REACTIONS_AUDIBLE = true;
+export const DEFAULT_REACTIONS_AUDIBLE = false;
```

Comment:
I think this wants to be `true` when you make the logic changes above.
```suggestion
export const DEFAULT_REACTIONS_AUDIBLE = true;
```

Reply:
renamed into `DEFAULT_REACTIONS_MUTED = false`